src/Connectors/test/ConnectorBase.Test/ConnectionStringConfigurationSourceTest.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the Apache 2.0 License.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Steeltoe.Connector.Test
+{
+    public class ConnectionStringConfigurationSourceTest
```

Comment:
Purpose of this class?

Reply:
I think I pushed contents to this file while you were reviewing